### DISCUSSION TITLE
This video is either unavailable or not supported in this browser. (Error #4000)

### DISCUSSION THREAD

**Discussion by g0t0wasd** [qutebrowser/qutebrowser#5713] [Q&A] [answered]
Whenever I try to reach twitch I'm getting error:

This video is either unavailable or not supported in this browser. (Error #4000)

for any twitch video I'm trying to watch. Has anyone experienced similar issue?

**Comment by mschilli87** [comment]
What OS are you on? How did you install qutebrowser? Can you share a link where you get that error that does nor require an account? I clicked on random videos on twitch.com and they all worked.

**Reply by g0t0wasd** [reply]
Ubuntu 20.04.1, installed with virtualenv. When I try to run any random video on twitch, it gives either:

The video download was cancelled. Please try again. (Error #1000)
or 
This video is either unavailable or not supported in this browser. (Error #4000)

Could it be related to adblock?

**Comment by The-Compiler** [comment, marked as answer]
As pointed out in the install instructions and the `mkvenv.py` script, if you install via virtualenv you don't get any support for proprietary codecs.

**Reply by GeekPatriot** [reply]
I have the same problem. I can't watch live streams but I can watch videos just fine. I always get the same error "This video is either unavailable or not supported in this browser. (Error #4000)". I'm on windows 10 and I installed qute browser Windows 64bit installer v1.14.1. It works on other browsers.

**Reply by toofar** [reply]
As per the FAQ https://qutebrowser.org/doc/install.html#_prebuilt_binaries the windows builds don't ship with proprietary codecs, you would have to build QtWebengine yourself. There are instructions on the Qt website but it is probably a rather involved process.

**Reply by GeekPatriot** [reply]
Yeah it's way above my knowledge. Shame because I really like the browser. Thanks for the reply!
